usage/app/lib/Config.scala

```diff
@@ -27,6 +29,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
   val capiPreviewUser = properties("capi.preview.user")
   val capiPreviewPassword = properties("capi.preview.password")
 
+  val topicArn = properties("sns.topic.arn")
```

Comment:
Does it need adding to the `dot-properties` dev script too? Presumably already added to CF?